src-tauri/src/main.rs

```diff
@@ -83,6 +83,7 @@ async fn connect_to_serial_port(
 ) -> Result<(), CommandError> {
     let mut connection = SerialConnection::new();
     let new_device = mesh::device::MeshDevice::new();
+    let graph = mesh::device::MeshGraph::new();
```

Comment:
I might place this in managed state per our discussion, which would allow graph modification from within commands

Reply:
Added an arc/mutex for the graph controlled by the app, unlocking it in `connect_to_serial_port`. Passed down the graph by reference through the `MeshDevice` functions.